### PR TITLE
task_2

### PR DESCRIPTION
**Pull request by maksim-kashytski**

### CODE REVIEW

File: task2.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        .finally(() => console.log('The text file is prepared for writing'));
+
+    csv({ output: 'line' })
+            .fromFile(CSV_File_Path)
+            .subscribe(
+                (csvLine) => appendFile(TXT_File_Path, csvLine + '\n', errorHandler),
+                errorHandler,
+                () => console.log('Successful')
+            );
+};
```
Comment (Taximu) [inline task2.js]: Please read the task again and introduce necessary changes. Well, actually, you didn't complete task 1.2

**Review by Taximu — CHANGES_REQUESTED** [review]
Please see comment

File: task2.js (hunk 1 above)
Comment (maksim-kashytski) [reply task2.js]: Updated

**Review by Taximu — CHANGES_REQUESTED** [review]
For better readability, please extract streams into separate variables and create handler for error cases and your pipeline can be written in one string.

File: task2.js [hunk 2]
```diff
@@ -1,26 +1,19 @@
...
+pipeline(
+    fs.createReadStream(CSV_File_Path),
+    csv(),
+    fs.createWriteStream(TXT_File_Path),
+    (err) => {
+        if (err) {
+            console.error('Failed', err);
+        } else {
+            console.log('Succeeded');
+        }
```
Comment (Taximu) [inline task2.js]: For better readability, please extract streams into separate variables and create handler for error cases and your pipeline can be written in one string.

**Review by Taximu — CHANGES_REQUESTED** [review]

File: task2.js (hunk 2 above)
Comment (maksim-kashytski) [reply task2.js]: Updated

Comment (Taximu) [reply task2.js]: Very well!

**Review by Taximu — APPROVED** [review]

**Pull request merged**